database/seeds/AdvertisementsSeeder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class AdvertisementsSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        for ($i=0; $i < 100; $i++) {
```

Comment:
you are making insert in loops. that means 100 mysql requests. Please build 1 array of 100 items and make ONE insert with using of this array

Reply:
Redid as a factory